build.gradle

```diff
@@ -97,6 +98,9 @@ ext {
 
   userMaxForks = project.hasProperty('maxParallelForks') ? maxParallelForks.toInteger() : null
 
+  userFlakyRetries = project.hasProperty('flakyRetries') ? flakyRetries.toInteger() : 1
+  userMaxTestFailures = project.hasProperty('maxTestFailures') ? maxTestFailures.toInteger() : 5
```

Comment:
How about maxTestRetries and maxTestRetryFailures? Also, we need to update the Readme to mention these.

Reply:
Ok